phpunit.xml

```diff
@@ -1,4 +1,4 @@
-<phpunit bootstrap="vendor/autoload.php">
+<phpunit bootstrap="src/autoload.php">
```

Comment:
Could you explain why this change was required? My guess just from looking at the code and PR description would be that the one `vendor/` is automatically generated and uses relative paths, while the one in `src/` is something we control (hence why it's tracked).

Reply:
Yeah – for sure. Using the `src/autoload.php` file in the bootstrap ensures we won't accidentally forget to add a file to it. If we go to run a test, and we didn't add the file to the manual autoloader, it'll error out saying "Class not found" or something similar. 

This gets us a little bit more safety for the less-commonly used approach.